contracts/interfaces/vesper/IVesperPool.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.28;
+
+import "./IGovernable.sol";
+import "./IPausable.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IVesperPool is IGovernable, IPausable, IERC20 {
```

Comment:
I'm not a fan of having huge interfaces, but I was going with what I assumed we want to do, which is include full interfaces of the contracts we interact with. We could cut out 80% of functions from the most of them. Wdyt?

Reply:
Yeah it's a tough one. 
If we need an interface then I think it's great to have everything on there, so it's complete and we can use whatever we need. I think that generally this is a good option when developing to create the interfaces first, at that stage you can't be sure what you need. It also might be beneficial now we're working with AI more as the AI can get more context about the protocol we're working with.
However if you keep following that route with chained imports you can end up going down some crazy routes and we need to stop it somewhere.

I started doing full implementations with the exception of imports that we don't already have or don't need.
Yes I guess we could cut down the interfaces with only the functions we need once we are done, but it doesn't impact gas or anything, only really the line count of the repo. And if we did need to go back and make changes it would save the double work of getting the interface again (although with AI that's also super easy now anyway).